On Windows make stops with "cp: cannot stat '/home/zmtemp/.local/bin/hie': No such file or directory

Windows 10
stack Version 1.7.0.3,  x86_64 hpack-0.28.2
haskell-ide-engine repo has been cloned recently

After some compilation and haddock folder access errors finally the first phase (stack-8.0.2.yaml) compiled but make couldn't continue to the second phase with this error (sorry for bad characters):

```
[0mhaskell-ide-engine-0.1.0.0: copy/register[0m
[0mCompleted 22 action(s).[0m
[0mLog files have been written to: C:\Users\zmtemp\Desktop\haskell\packages\haskell-ide-engine\.stack-work\logs\[0m
[0mCopying from C:\Users\zmtemp\Desktop\haskell\packages\haskell-ide-engine\.stack-work\install\71c8e9f1\bin\hie.exe to C:\Users\zmtemp\AppData\Roaming\local\bin\hie.exe[0m
[0m[0m
[0mCopied executables to C:\Users\zmtemp\AppData\Roaming\local\bin:[0m
[0m- hie.exe[0m
cp: cannot stat '/home/zmtemp/.local/bin/hie': No such file or directory
make: *** [Makefile:4: build] Error 1
```
After I created /home/zmtemp/.local/bin/hie directory in msys2 terminal, the error message changed to this:
```
cp: -r not specified; omitting directory '/home/zmtemp/.local/bin/hie'
make: *** [Makefile:4: build] Error 1
```

--- a/Makefile
+++ b/Makefile
@@ -1,4 +1,5 @@
 BASEDIR=$(CURDIR)
+STACKLOCALBINDIR != stack path --local-bin
 
 build: hie-8.2.1 hie-8.2.2 hie-8.4.2
 # build: hie-8.0.2 hie-8.2.1 hie-8.2.2 hie-8.4.2
@@ -22,29 +23,28 @@ build-all: | build build-docs
 .PHONY: build
 
 # ------------------------------------------------------
-
 hie-8.0.2:
-	stack --stack-yaml=stack-8.0.2.yaml install                  \
-		&& cp ~/.local/bin/hie ~/.local/bin/hie-8.0      \
-		&& cp ~/.local/bin/hie-8.0.2 ~/.local/bin/hie-8.2
+	stack --stack-yaml=stack-8.0.2.yaml install                                \
+		&& cp '$(STACKLOCALBINDIR)/hie' '$(STACKLOCALBINDIR)/hie-8.0'      \
+		&& cp '$(STACKLOCALBINDIR)/hie-8.0.2' '$(STACKLOCALBINDIR)hie-8.2'
 .PHONY: hie-8.0.2
 
 hie-8.2.1:
-	stack --stack-yaml=stack-8.2.1.yaml install                  \
-		&& cp ~/.local/bin/hie ~/.local/bin/hie-8.2.1      \
-		&& cp ~/.local/bin/hie-8.2.1 ~/.local/bin/hie-8.2
+	stack --stack-yaml=stack-8.2.1.yaml install                                \
+		&& cp '$(STACKLOCALBINDIR)/hie' '$(STACKLOCALBINDIR)/hie-8.2.1'    \
+		&& cp '$(STACKLOCALBINDIR)/hie-8.2.1' '$(STACKLOCALBINDIR)/hie-8.2'
 .PHONY: hie-8.2.1
 
 hie-8.2.2:
-	stack --stack-yaml=stack-8.2.2.yaml install                  \
-		&& cp ~/.local/bin/hie ~/.local/bin/hie-8.2.2      \
-		&& cp ~/.local/bin/hie-8.2.2 ~/.local/bin/hie-8.2
+	stack --stack-yaml=stack-8.2.2.yaml install                                \
+		&& cp '$(STACKLOCALBINDIR)/hie' '$(STACKLOCALBINDIR)/hie-8.2.2'    \
+		&& cp '$(STACKLOCALBINDIR)/hie-8.2.2' '$(STACKLOCALBINDIR)/hie-8.2'
 .PHONY: hie-8.2.2
 
 hie-8.4.2:
-	stack --stack-yaml=stack.yaml install                  \
-		&& cp ~/.local/bin/hie ~/.local/bin/hie-8.4.2      \
-		&& cp ~/.local/bin/hie-8.4.2 ~/.local/bin/hie-8.4
+	stack --stack-yaml=stack.yaml install                                      \
+		&& cp '$(STACKLOCALBINDIR)\hie' '$(STACKLOCALBINDIR)\hie-8.4.2'    \
+		&& cp '$(STACKLOCALBINDIR)\hie-8.4.2' '$(STACKLOCALBINDIR)\hie-8.4'
 .PHONY: hie-8.4.2
 
 # ------------------------------------------------------

